bin/compile

```diff
@@ -51,7 +51,7 @@ for BUILDPACK in $(cat $1/.buildpacks); do
     # Ensure that these files exist.
     chmod -f +x $dir/bin/{detect,compile,release} || true
 
-    framework=$($dir/bin/detect $1/$subdir)
+    framework=$(bash $dir/bin/detect $1/$subdir)
```

Comment:
The hashbangs specify which shell to use for this, is it necessary here?

https://github.com/negativetwelve/heroku-buildpack-subdir/blob/master/bin/detect#L1

Reply:
You're right, this isn't needed. I'll remove it.